transactions/burn-flow/2022/sep-21/README.md

```diff
@@ -1,10 +1,17 @@
 # Burn Flow - Sep 21
 
 The Flow Service account will burn 38,357,187 FLOW after it has been
-transferred from the Genesis Token Account. These tokens are accounted for as
-part of the FlowToken.totalSupplybut are *not currently considered* as part of
+transferred from the Genesis Token Account. These tokens are "Bonus Tokens" and 
```

Comment:
can we remove this transfer note? It's confusing/needlessly concerning

Reply:
Agreed someone can ask why not burning 65m? Did they earned rewards? Where is the rest? 

I suggest keep transparency at minimum level ( foundation is good example for that ) 

Flow service account will burn 38M to promote Flow ecosystem is great!